tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -896,7 +896,64 @@ static py::object GetValueOrNone(py::kwargs &kwargs, const string &key) {
 	return py::none();
 }
 
+void AcceptableCSVOptions(const py::kwargs &kwargs) {
+	// List of strings to match against
+	const unordered_set<string> valid_parameters = {"header",
```

Comment:
Having this list of parameters effectively duplicated seems error-prone when we will (inevitably) add more parameters.

Could we perhaps rewrite the `ReadCSV` function below to iterate over the `kwargs`, e.g. something like:

```cpp
for(auto &arg : kwargs) {
   auto &arg_name = arg.first;
   if (arg_name == "header") {
        ...
   } else if (arg_name == "compression") {
        ...
   } else {
        throw Exception("Unrecognized parameter X");
   }
}
```